indigo/distribution.yaml

```diff
@@ -4338,10 +4338,11 @@ repositories:
       - libmavconn
       - mavros
       - mavros_extras
+      - test_mavros
```

Comment:
I think to release it, because it stores hand-test for simulators (APM SITL and PX4 ROS SITL).